packages/app-project/src/shared/components/ProjectStatistics/ProjectStatisticsContainer.spec.js

```diff
@@ -1,62 +1,58 @@
-import { shallow } from 'enzyme'
-
-import ProjectStatistics from './ProjectStatistics'
-import { ProjectStatisticsContainer } from './ProjectStatisticsContainer'
-
-let wrapper
-let projectStatisticsWrapper
-
-const CLASSIFICATIONS = 1
-const COMPLETED_SUBJECTS = 2
-const ROUTER = {
-  asPath: '/foo/bar',
-  query: {
-    owner: 'foo',
-    project: 'bar'
-  }
+import { render, screen, waitFor } from '@testing-library/react'
+import { within } from '@testing-library/dom'
+import { composeStory } from '@storybook/react'
+import Meta, { ProjectStatistics } from './ProjectStatisticsContainer.stories.js'
+import { ProjectStatisticsContainerMock, ProjectStatisticsContainerRouterMock } from './ProjectStatisticsContainer.mock'
```

Comment:
This is an okay solution, but I recommend importing mocks for use in Stories if you're going to import the Stories for use in this spec file anyway. For instance, there's no router mock in ProjectStatisticsContainer.stories, so the link to View More Stats is incorrect when viewing app-project's Storybook. If the router mock is used in the story file, then you can use the full-formed story components in the spec file, and you've got a consistent router mock for both test environments.

An example of this pattern is used in WorkflowAssignmentModal. There's a mock [router](https://github.com/zooniverse/front-end-monorepo/blob/a73489fe5f1fbca1778850bff19abbbcc6ecba30/packages/app-project/src/screens/ClassifyPage/components/WorkflowAssignmentModal/WorkflowAssignmentModal.stories.js#L13), mock [store](https://github.com/zooniverse/front-end-monorepo/blob/a73489fe5f1fbca1778850bff19abbbcc6ecba30/packages/app-project/src/screens/ClassifyPage/components/WorkflowAssignmentModal/WorkflowAssignmentModal.stories.js#L46), and then WorkflowAssignmentModal's spec file uses the fully formed [Story](https://github.com/zooniverse/front-end-monorepo/blob/a73489fe5f1fbca1778850bff19abbbcc6ecba30/packages/app-project/src/screens/ClassifyPage/components/WorkflowAssignmentModal/WorkflowAssignmentModal.spec.js#L8) without additional mocks.

Reply:
Thanks for the links to the way this is done in WorkflowAssignmentsModal - was very helpful. I still like keeping the separate mock file so that the spec can reference the data instead of hardcoding it / creating brittleness. I was also able to fix the link and test for it in appropriately in the spec file.